powerdns/admin.py

```diff
@@ -388,15 +394,18 @@ def get_form(self, request, obj=None, **kwargs):
             form.base_fields['domain'].initial = self.from_object.domain
         return form
 
-admin.site.register(Domain, DomainAdmin)
-admin.site.register(Record, RecordAdmin)
-admin.site.register(SuperMaster, SuperMasterAdmin)
-admin.site.register(DomainMetadata, DomainMetadataAdmin)
-admin.site.register(CryptoKey, CryptoKeyAdmin)
-admin.site.register(DomainTemplate, DomainTemplateAdmin)
-admin.site.register(RecordTemplate, RecordTemplateAdmin)
-admin.site.register(Authorisation, AuthorisationAdmin)
-admin.site.register(DomainRequest, DomainRequestAdmin)
-admin.site.register(RecordRequest, RecordRequestAdmin)
-admin.site.register(DeleteRequest, DeleteRequestAdmin)
-admin.site.register(TsigKey)
+admin_site.register(Domain, DomainAdmin)
+admin_site.register(Record, RecordAdmin)
```

Comment:
what about reversing urls to the "old" admin? it had prefix "admin" before and now it will probably not work (it'll try to point to the new admin, ex. https://github.com/allegro/django-powerdns-dnssec/blob/develop/powerdns/models/powerdns.py#L141)